cpp/src/ray/test/api_test.cc

```diff
@@ -156,3 +156,55 @@ TEST(RayApiTest, CompareWithFuture) {
   EXPECT_EQ(rt3, 2);
   t.join();
 }
+
+RAY_REGISTER(Plus1);
+
+TEST(RayApiTest, RayRegister) {
```

Comment:
Can we put it into a standalone test file before this file is getting too large? Do you plan to put all tests into one single file?

Reply:
Good, put it into another file would be better.

Reply:
Moved the test code into api_v2_test.cc.

Reply:
Maybe put it into `ray_register_test` and split one big test case into some smaller test cases.

Reply:
For example, you can put `/// Not exist function.` and `/// Arguments not match.` into two different test cases.